db/migrate/20190818163655_rename_table.rb

```diff
@@ -0,0 +1,10 @@
+class RenameTable < ActiveRecord::Migration[5.2]
+  def change
+
+    def self.up
+      rename_table :admins_book_clubs, book_clubs_admins
```

Comment:
There is a typo: 
rename_table :admins_book_clubs, book_clubs_admins
should be:
rename_table :admins_book_clubs, **:**book_clubs_admins